README.md

```diff
@@ -4,6 +4,12 @@ This is the examples repository for the [strongDM Python SDK](https://github.com
 
 ---
 > **NOTE:**  
-> This repository is currently under construction. The content is expected to change in the coming weeks.
-
+> To increase flexibility when managing thousands of Resources, Role Grants have
```

Comment:
Same comments as on the Terraform repo about a more generic statement here than "thousands of Resources", and down below about granting access to resources. Also, static access rules don't only exist for backwards compatibility, afaik, but maybe if that's the primary use case that's enough here, since this isn't full documentation.